### PR TITLE
chore: add issue form for car

### PR DESCRIPTION
**Pull request by skipishere**
Here's an initial attempt at creating an issue form to get information from users using the current new issue template as the basis.

image

I am in two minds if this should be a bug form rather than a general issue one, given the questions asked/input required?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue_car.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+name: Telsa Car Issue
+description: Create a report to help us improve
+title: "Car Issue: "
+labels: ["triage", "car"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Before you open a new issue, search through the existing issues to see if others have had the same problem.
```
Comment (alandtse) [inline .github/ISSUE_TEMPLATE/issue_car.yml:9]: I've tended to also mention "closed" issues for searching too.

**Review by alandtse — APPROVED** [review]
I'm ok starting with a general issue and then we can choose it later if there's a need. Typically it's bug vs enhancement request.

File: .github/ISSUE_TEMPLATE/issue_car.yml (hunk 1 above)
Comment (alandtse) [reply .github/ISSUE_TEMPLATE/issue_car.yml:9]: @skipishere just wanted to confirm if you wanted to add a statement about searching closed issues too. If not, I can just merge it but wasn't sure what you were planning.

Comment (skipishere) [reply .github/ISSUE_TEMPLATE/issue_car.yml:9]: Sorry not had a lot of time, will try and do that today 👍

Comment (skipishere) [reply .github/ISSUE_TEMPLATE/issue_car.yml]: Sorry took longer than planed to get back to this, happy to merge this whenever you are

**Review by alandtse — APPROVED** [review]

**Pull request merged**